tests/integration/test_utils_archive_readers.py

```
@@ -295,6 +295,40 @@ def test_raises_import_error_when_py7zr_unavailable(self, tmp_path: Path) -> Non
+        from file_organizer.utils.readers import archives
+        from file_organizer.utils.readers.archives import read_7z_file
+
+        fake_file = SimpleNamespace(
+            filename="sample.txt",
+            compressed=512,
+            uncompressed=1024,
+        )
+        fake_archive = MagicMock()
+        fake_archive.list.return_value = [fake_file]
+        fake_archive.password_protected = False
+        fake_archive.__enter__ = lambda s: fake_archive
```

Ruff (0.15.9)

[warning] Unused lambda argument: `s`

(ARG005)